tools/travis-test.sh

```diff
@@ -8,7 +8,10 @@ export NPY_NUM_BUILD_JOBS=2
 # setup env
 if [ -r /usr/lib/libeatmydata/libeatmydata.so ]; then
   # much faster package installation
-  export LD_PRELOAD=/usr/lib/libeatmydata/libeatmydata.so
+  export LD_PRELOAD='/usr/lib/libeatmydata/libeatmydata.so'
+elif [ -r /usr/lib/*/libeatmydata.so ]; then
+  # much faster package installation
+  export LD_PRELOAD='/usr/$LIB/libeatmydata.so'
```

Comment:
preload understands the special LIB variable, but we have to only enable it if it is available as our build will fail if gcc emits stderr (a bug we should fix) so I use a wildcard in the check, a little clunky but works